src/controllers/trust.details.controller.ts

```diff
@@ -0,0 +1,113 @@
+import { NextFunction, Request, RequestHandler, Response } from 'express';
+import * as config from '../config';
+import { logger } from '../utils/logger';
+import { getApplicationData, setExtraData } from '../utils/application.data';
+import { TrustMapperToPageService, trustMapperToPageService } from '../service/TrustMapperToPageService';
+import { ApplicationData } from '../model/application.model';
+import * as PageModel from '../model/Trust/page.model';
+import { trustMapperToSessionService, TrustMapperToSessionService } from '../service/TrustMapperToSessionService';
+import { Trust } from 'model/trust.model';
+
+interface TrustDetailsController {
+  get: RequestHandler;
+  post: RequestHandler;
+}
+
+const TrustDetailsControllerFactory = (
+  mapperToPageSrv: TrustMapperToPageService,
```

Comment:
Would be inclined to not shorten these variable names. Just have `mapperToPageService` and `mapperToSessionService`.